channel/adjudicator.go

```diff
@@ -36,30 +36,52 @@ type (
 	// AdjudicatorEvents. Those events might be triggered by a Register or
 	// Progress call on the adjudicator from any channel participant.
 	Adjudicator interface {
-		// Register should register the given ledger channel state on-chain.
-		// If the channel has locked funds into sub-channels, the corresponding
-		// signed sub-channel states must be provided.
+		Registerer
+		Withdrawer
+		Progresser
+		AdjudicatorEventSubscriber
```

Comment:
Rename to `EventSubscriber` or simply `Subscriber`?

Reply:
Renamed to `EventSubscriber` in this [force push](https://github.com/hyperledger-labs/go-perun/compare/e86f82174c996ed2a88b3d4e9458872444df4563..71f5bc88d52ffd45b33b957d88780009680a6e6c).